README.md

```diff
@@ -94,22 +94,61 @@ TODO: more integrations will be added below as the [Hydra GSoC project progresse
+In hydra.js, it initiates the Bridge, and starts listening to the token response from the Hydra. It also have the method `(_getTokenFromCookies)` to fetch the token from the cookies and pass it to the integrator to use it in the `ploneClient.initialize()`.
 
-**Note**: If you are not logged in at adminUI, then `get_token()` will return empty string and client will fetch only public content.
-  ```js
-  // After Bridge is initialized
-  import ploneClient from '@plone/client';
-  import { get_token } from './hydra.js';
+Integrate your frontend:
+
+- Add 'hydra.js` in your frontend.
+- Initialize the Bridge using `initBridge` method provided by './hydra.js', use 'https://hydra.pretagov.com' for option `adminOrigin` to tryout demo.
+- Use the `getToken()` method provided by './hydra.js' to access the token. Use this in your ploneClient inctance.
+- At [Volto-Hydra demo](https://hydra.pretagov.com/) type in your hosted frontend url to preview public content and login to see the private pages.
 
-  const hydra_token = get_token(); // either be 'auth_token' or '', based on weather you are logged in or not at adminUI.
+Example Usage:
+```js
+// nextjs 14
+import ploneClient from "@plone/client";
+import { useQuery } from "@tanstack/react-query";
+import { initBridge } from "@/utils/hydra";
+
+export default function Blog({ params }) {
+  const bridge = initBridge("http://localhost:3000"); // Origin of your local Volto-Hydra
+  const token = bridge._getTokenFromCookie();
   const client = ploneClient.initialize({
-    apiPath: 'http://localhost:8080/Plone',
-    token: hydra_token,
+    apiPath: "http://localhost:8080/Plone/", // Plone backend
+    token: token,
   });
-  ```
-TODO: will be implemented before the end of this iteration
+  const { getContentQuery } = client;
+  const { data, isLoading } = useQuery(getContentQuery({ path: '/blogs' }));
+
+  if (isLoading) {
+    return <div>Loading...</div>;
+  }
+  return (
+    <div> {data.title}</div>
+  )
+}
+```
+TODO: Once demo is deployed,in the above example replace origin and plone backend url with demo server.
+
+Reference Issue: [#6](https://github.com/collective/volto-hydra/issues/6)
+
+#### Add a new Page
+
+You can click on Add page, type in Title and click save and you will be redirected to the edit mode then you can edit the page's content. In the edit mode, you can change title of the page from the sidebar and save it to see the title of the page updated.
+
+Reference Issue: [#7](https://github.com/collective/volto-hydra/issues/11)
+
+#### Remove a page
+
+You can go to content view, to delete single or multiple pages at once.
+
+#### Edit a page
+
+You can click on edit while previewing that page to enter see sidebar appear. You can change title of the page.
+
+TODO: editing of Blocks/contents of a page is not implemented yet.
+
+Reference Issue: [#13](https://github.com/collective/volto-hydra/issues/13)
```

Comment:
I don't think its important to list out this much details. You can just say

---
Now your editors can navigate the site within the editor or in the frontend displayed in the middle of the screen, add, remove and do normal plone toolbar functions as well as edit a page metadata via the sidebar.